app/ServiceProvider/ApiProvider.php

```diff
@@ -60,6 +61,7 @@ public function register(Container $container)
             ->withObject(new ProjectApi($container))
             ->withObject(new ProjectPermissionApi($container))
             ->withObject(new SubtaskApi($container))
+            ->withObject(new SubtaskTimeTracking($container))
```

Comment:
The class name should have the suffix "Api" like the other ones to be consistent.